packages/vscode.vm/vscode.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>vscode.vm</id>
-    <version>1.85.2.20240304</version>
+    <version>1.85.2.20240410</version>
```

Comment:
This dependency is wrongly updated, likely related to https://github.com/mandiant/VM-Packages/issues/972